src/main/java/com/woowacamp/soolsool/core/statistics/dto/response/LiquorSalePriceResponse.java

```diff
@@ -0,0 +1,29 @@
+package com.woowacamp.soolsool.core.statistics.dto.response;
+
+import com.woowacamp.soolsool.core.liquor.domain.Liquor;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class LiquorSalePriceResponse {
+
+    private Long id;
+    private String name;
+    private String brew;
+    private String region;
+    private String imageUrl;
+    private Long totalSalePrice;
+
+    public static LiquorSalePriceResponse from(final Liquor liquor, final Long totalSalePrice) {
```

Comment:
```suggestion
    public static LiquorSalePriceResponse from(
        @NonNull final Liquor liquor,
        @NonNull final Long totalSalePrice
    ) {
```

이건 지금까지 패턴에는 없던거지만 이렇게 해서 NPE를 메서드 수준에서 방지하는건 어떤가요? 지금도 `from()` 안에서 NPE가 뜨지만 해당 NPE는 실제로 참조해서 뜨는 NPE이고, `@NonNull`의 경우 lombok에 의해 발생하는 NPE라서 보다 빨리 예외를 알 수 있을 것 같아요. 

적용하라는 건 아니지만 생각이 궁금합니다!

Reply:
of

Reply:
좋아용~ 두분다 감솨합니다^^